README.md

```diff
@@ -89,6 +89,7 @@ Then configure the channels as explained in the [Configure youtube-dl](https://g
 | `youtubedl_watchlater` | `true` (`false`) | If you want to download your Watch Later playlist. Authentication is required.
 | `youtubedl_interval` | `1h` (`3h`) `12h` `3d` | If you want to change the default download interval. This can be any argument compatible with [gnu sleep](https://github.com/tldr-pages/tldr/blob/main/pages/linux/sleep.md).  Be mindful that if you download too frequently, you risk getting ip-banned by YouTube.<br />1 hour, (3 hours), 12 hours, 3 days.
 | `youtubedl_quality` | `720` (`1080`) `1440` `2160` | If you want to change the default download resolution.<br>720p, (1080p), 1440p, 4k.
+| `post_processing_script` | `/scripts/cleanup.bash` (`''`) | If you want to run a script after downloading has completed set this to the path to the script you want to run. Script must be available in the container.
```

Comment:
I would like you to remove this.

We should simply make it discover and run both post & pre-execution scripts if the files exist, no need to make the name user definable.

Do you have any issues with me putting the scripts in /config/?

Reply:
That makes sense, keeping them in the already mounted directory seems reasonable so no additional mounting is needed